python/cuvs/cuvs/neighbors/vamana/vamana.pxd

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# cython: language_level=3
+
+from libc.stdint cimport int32_t, uint32_t, uintptr_t
+from libcpp cimport bool
+
+from cuvs.common.c_api cimport cuvsError_t, cuvsResources_t
+from cuvs.common.cydlpack cimport DLDataType, DLManagedTensor
+from cuvs.distance_type cimport cuvsDistanceType
+
+
+cdef extern from "cuvs/neighbors/vamana.h" nogil:
+
+    ctypedef struct cuvsVamanaIndexParams:
+        cuvsDistanceType metric
+        uint32_t graph_degree
+        uint32_t visited_size
+        uint32_t vamana_iters
```

Comment:
Same as previous comment - is a `float` value now

Reply:
Done!